doc/sphinx/addendum/extraction.rst

```diff
@@ -215,14 +224,14 @@ code elimination performed during extraction, in a way which
 is independent but complementary to the main elimination
 principles of extraction (logical parts and types).
 
-.. cmd:: Extraction Implicit @qualid [ {+ @ident } ]
+.. cmd:: Extraction Implicit @qualid [ {* {| @ident | @int } } ]
 
    This experimental command allows declaring some arguments of
-   :token:`qualid` as implicit, i.e. useless in extracted code and hence to
-   be removed by extraction. Here :token:`qualid` can be any function or
-   inductive constructor, and the given :token:`ident` are the names of
-   the concerned arguments. In fact, an argument can also be referred
-   by a number indicating its position, starting from 1.
+   :token:`qualid` as implicit, i.e. useless in extracted code and so
+   should be removed during extraction. Here :token:`qualid` can be
```

Comment:
I found the previous wording "to be removed" clearer than "declaring as (...) should be removed". Maybe you can propose something entirely different?

Reply:
How about:

```
   Declares some arguments of
   :token:`qualid` as implicit, meaning that they are useless in extracted code.
   The extracted code will omit these arguments.
```

Reply:
Yes, that's better.